makefiles/stdio.inc.mk

```diff
@@ -29,6 +30,15 @@ ifneq (,$(filter stdio_ethos,$(USEMODULE)))
   USEMODULE += stdio_uart
 endif
 
+ifneq (,$(filter stdio_nimble,$(USEMODULE)))
+  USEPKG += nimble
+  USEMODULE += isrpipe
+  USEMODULE += nimble_svc_gap
+  USEMODULE += nimble_svc_gatt
+  USEMODULE += ztimer_msec
+  FEATURES_REQUIRED += periph_uart
```

Comment:
Condtionally include?

Reply:
`FEATURES_REQUIRED += periph_uart` is now conditionally included by `stdio_nimble_debug`